flint/summary.py

```diff
@@ -126,11 +149,16 @@ def add_ms_summaries(
     hour_angles = centre_altaz.az.to(u.hourangle)
     elevations = centre_altaz.alt.to(u.deg)
 
+    # The INSTRUMENT_RECEPTOR_ANGLE comes from fixms and is
```

Comment:
Once it has been added to the field summary, then it is passed through to the linmos tasks.